ModelTraining/CosmosData/script.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.Azure.Cosmos;
+
+string endpoint = "https://aifororcasmetadatastore.documents.azure.com:443/";
+string key = "[INSERT PRIMARY KEY HERE]";
+
+CosmosClient client = new CosmosClient(endpoint, key);
+AccountProperties account = await client.ReadAccountAsync();
+
+// Sanity check
+Console.WriteLine($"Account Name:\t{account.Id}");
+
+// Get the database
+Database database = await client.CreateDatabaseIfNotExistsAsync("predictions");
+Container container = await database.CreateContainerIfNotExistsAsync("metadata", "/source_guid");
+
+string sql = "SELECT m.id, m.modelId, m.audioUri, m.imageUri, m.reviewed, m.timestamp, m.whaleFoundConfidence, m.location.id AS location_id, m.location.name AS location_name, m.location.longitude AS location_long, m.location.latitude AS location_lat, m.source_guid, p.id AS prediction_id, p.startTime AS prediction_startTime, p.duration AS prediction_duration, p.confidence AS prediction_confidence FROM metadata m JOIN p IN m.predictions";
+QueryDefinition query = new (sql);
+
+QueryRequestOptions options = new ();
+options.MaxItemCount = 50;
```

Comment:
If the developer is using Visual Studio, they can put the url and key in their User Secrets so that it does not go into the checked in code. Otherwise they will need to put it into appsettings.Development.json file and make sure that file does not get checked in as part of the pull request.

Reply:
I thought it was a console application, hence my suggestion. User secrets/appsettings.Development.json are generally used for web apps.